ci/before_deploy.bash

```diff
@@ -38,6 +38,11 @@ pack() {
     cp LICENSE-MIT "$tempdir/$package_name"
     cp LICENSE-APACHE "$tempdir/$package_name"
 
+    # various autocomplete
+    cp target/"$TARGET"/release/build/"$PROJECT_NAME"-*/out/"$PROJECT_NAME".bash "$tempdir/$package_name/autocomplete/${PROJECT_NAME}.bash-completion"
```

Comment:
We need to make sure that the target directory exists.

Reply:
Also, it would be great to also do this for Windows deployments (in the `before_deploy.ps1` script). I think we don't need a subfolder in case of the Windows archive.

Reply:
I'm not familiar with how it should work on Windows. 

Should it be something like this ?
```ps1
Copy-Item "$SRC_DIR\target\release\build\bat-*/out/_bat.ps1" '.\autocomplete'
```